core/lib/config/src/configs/en_config.rs

```diff
@@ -0,0 +1,40 @@
+use std::num::{NonZeroU32, NonZeroU64, NonZeroUsize};
+
+use serde::Deserialize;
+use zksync_basic_types::{
+    commitment::L1BatchCommitmentMode, url::SensitiveUrl, L1ChainId, L2ChainId,
+};
+
+#[derive(Debug, Clone, PartialEq, Deserialize)]
+pub struct Pruning {
+    pub enabled: bool,
+    pub chunk_size: Option<u32>,
+    pub removal_delay_sec: Option<NonZeroU64>,
+    pub data_retention_sec: Option<u64>,
+}
+
+#[derive(Debug, Clone, PartialEq, Deserialize)]
+pub struct SnapshotRecovery {
+    pub enabled: bool,
+    pub postgres_max_concurrency: Option<NonZeroUsize>,
+    pub tree_chunk_size: Option<u64>,
+}
+
+/// Temporary config for initializing external node, will be completely replaced by consensus config later
```

Comment:
This comment looks bogus. From the fields below, only first 3 look like something agreed upon by the network.

Reply:
So, the first 3 will be moved to the genesis over l1 or consensus and the next 2 fields are gonna move to the nodes in consensus config.

I'd like to keep it as is for the sake of simplicity